### ISSUE TITLE
qti2gift?

### ISSUE THREAD

**Issue by dirkcgrunwald** [csev/gift2qti#8]
We have someone who authored questions using Coursera's tools and now needs to import them into Moodle.

Is there a similar tool to do the reverse translation?

**Comment by csev** [comment]
Dave, I have done this myself but have not built / released any software to do this.  I write some Python to take the Coursera format - not the QTI format the Coursera text format and turn that into GIFT.  I had to do some tweaking after the conversion and I did not keep working once my conversion worked well enough along with my hand-hacking.  It is in uncommented Python and sitting in one of my private repos.  If you were interested and willing to improve it - we could extract it and make a separate github repo for it.  I don't have time to fix it for you if your use cases are different - but perhaps you could take what I built and improve on it.  Interested?

**Comment by dirkcgrunwald** [comment]
Yes, I'd be interested - I'm assuming the Coursera text format is the "word export" format like that below?

```
Question 1 - multiple choice, no shuffle
In which of the following categories do you think the decision tree approach lies in?
*A: It is a high variance method
Feedback: Your answer is correct. 
B: It is a high bias method
Feedback: Your answer is incorrect. The correct answer is: It is a high variance method. 
C: It is a low variance method
Feedback: Your answer is incorrect. The correct answer is: It is a high variance method. 
```
